ee/api/ee_event_definition.py

```diff
@@ -30,10 +34,15 @@ class Meta:
             "volume_30_day",
             "query_usage_30_day",
             "last_seen_at",
+            "verified_at",
         ]
 
     def update(self, event_definition: EnterpriseEventDefinition, validated_data):
         validated_data["updated_by"] = self.context["request"].user
+        if "verified" in validated_data and validated_data["verified"]:
+            validated_data["verified_by"] = self.context["request"].user
```

Comment:
- This could lead to the verified_by being overridden even if the event has already been verified.
- If the event was previously verified and you remove verification we should clear `verified_at` & `verified_by`

Having said the above, if we do plan to have full history on this soon as you suggested, we then might be better off removing all these.

Reply:
I'll move clearing logic in the model up into the serializer, which should take care of both points. I'd prefer to separate this feature from full history, but we can consider removing this once history is fully implemented